igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -91,18 +93,54 @@
     "hyp3 = sdk.HyP3(prompt=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f387a5e2-f145-41b8-8cb8-f8d29155fff0",
+   "metadata": {},
+   "source": [
+    "2. Select your data: Use ASF's [Vertex](https://search.asf.alaska.edu/) data search portal to find Sentinel-1 scenes your interesed in processing."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "739d2ebe-e14e-4998-a37e-f9b32d106902",
    "metadata": {},
    "outputs": [],
    "source": [
-    "project_name = 'edgecumbe_example'\n",
     "reference = 'S1B_IW_SLC__1SDV_20180628T151540_20180628T151607_011575_015476_4673'\n",
     "secondary = 'S1B_IW_SLC__1SDV_20190705T151547_20190705T151614_017000_01FFC4_D62F'"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d74b4574-0bcd-4ce7-8bda-cc2bb95f17ee",
+   "metadata": {},
+   "source": [
+    "3. Define your processing parameters: Determine the specific processing parameters for your InSAR product. This includes selecting the desired satellite data acquisitions, multilook, and whether to apply a water mask before unwrapping. You can read more about the available parameters within the [HyP3 SDK documentation](https://hyp3-docs.asf.alaska.edu/using/sdk_api/#hyp3_sdk.hyp3.HyP3.submit_insar_job)."
```

Comment:
This description includes "selecting the desired satellite data acquisitions", but it looks like those parameters aren't included in this code block, but rather in the previous one. Either move that reference to the previous cell block, or move the code with the scenes into this code block.

Reply:
Thanks for this catch!